tdrs-backend/docker-compose.yml

```diff
@@ -13,10 +13,15 @@ services:
   postgres:
     image: postgres:11.6
     environment:
-      - POSTGRES_USER=tdpuser
-      - POSTGRES_PASSWORD=something_secure
+      - PGDATA=/var/lib/postgresql/data/
       - POSTGRES_DB=tdrs_test
+      - POSTGRES_PASSWORD=something_secure
```

Comment:
Outside scope of this ticket, but should we create a follow-on ticket to not have a plaintext password in this file? Is there some way we can pass this in via env/cfg file, etc.? And at the same time make it a hash?

Reply:
This particular postgres is only used for local and CI so it doesn't necessarily need to be a secret. We could probably run postgres without a password locally too, that may be a simpler way to keep with the practice of not storing any plaintext passwords at all.

Reply:
Given that this is for local stack only, I think we could probably move to passwordless access to clean this up. However, I'm not sure for future testing if that might make testing deployed applications with a passworded postgres different enough to present some difficulty. I'll create a follow-on ticket since this is ultra-low priority.